tests/test_hotkeys.py

```
@@ -0,0 +1,620 @@
+        callback = Mock()
+        register_hotkey('a', callback)
+        register_hotkey('b', callback)
+        
+        clear_all_hotkeys()
+        
+        assert len(get_registered_hotkeys()) == 0
+    
+    @patch('agor.tools.hotkeys.KEYBOARD_AVAILABLE', True)
+    def test_start_hotkey_manager_global(self, mock_logger):
+        """Test global start_hotkey_manager function."""
+        manager = hotkey_manager()
```

Ruff (0.11.9)

Local variable `manager` is assigned to but never used

Remove assignment to unused variable `manager`

(F841)